### ISSUE TITLE
FaxVax UI spec v1

### ISSUE THREAD

**Issue by racheledelman** [codeforamerica/kansas-city-fellows#2]
* request form
  * child’s name
  * child’s DOB
  * parent’s name
  * parent’s contact info
   * email?
   * phone?
* add recipient
  * select school
   * dropdown?
   * predictive text?
* release agreement
  * release text with child’s name (pulled from above)
  * parent’s signature (type in)
* Submit button

For v2:
* error states
* manually add school (name + fax #)?
* feedback link

**Comment by lostmarinero** [comment]
I like this => Am setting up the api. Do you think the data validation should happen on frontend or backend? I was thinking frontend

**Comment by jessonawhim** [comment]
Thinking we would also add introductory content for context setting here

**Comment by racheledelman** [comment]
Good point, Jessica. Kevin, what do you think about using Typeform for this?

**Comment by jessonawhim** [comment]
for childs name, we may eventually need first and last separated depending on how they do the lookup, though i love not making them do that if they dont have to

**Comment by racheledelman** [comment]
Totally. But as long as we're faxing it to them to manually look up, I say single field.

**Comment by jessonawhim** [comment]
testing for rachels email

**Comment by jessonawhim** [comment]
second times a charm

**Comment by racheledelman** [comment]
Here is a wireframe of the above. My form labels and descriptions are very bad, so consider them placeholders:

![vf-wireframe](https://cloud.githubusercontent.com/assets/2693690/12600591/d030681e-c44f-11e5-91e1-d44377204523.png)
